### DISCUSSION TITLE
Explorer.exe crashing. r6030 crt not initialized.

### DISCUSSION THREAD

**Discussion by SD308** [valinet/ExplorerPatcher#1602] [General]
The issue I've had is that the app crashes explorer.exe a lot. Transferring files is the main task that causes it to crash. If I try to load explorer to see files, or transfer files, I will get the error: "runtime error explorer.exe r6030 crt not initialized". I am using the latest Windows 11 Pro, as well as the latest Explorer Patcher (since I tried to install it).

**Comment by pyrates999** [comment]
Duplicate of https://github.com/valinet/ExplorerPatcher/issues/1601
